enhancements/cluster-logging/forwarder-to-otlp.md

```diff
@@ -0,0 +1,204 @@
+---
+title: forwarder-to-otlp
+authors:
+  - "@jcantrill"
+reviewers:
+  - "@alanconway"
+  - "@cahartma"
+  - "@pavolloffay"
+  - "@periklis"
+  - "@xperimental"
+approvers:
+  - "@alanconway"
+api-approvers: 
+  - "@alanconway"
+creation-date: 2024-04-30
+last-updated: 2024-04-30
+tracking-link:
+  - "https://issues.redhat.com/browse/LOG-4225"
+see-also:
+  - "/enhancements//cluster-logging/cluster-logging-v2-apis.md"
+replaces: []
+superseded-by: []
+---
+
+# Log Forwarder to OTLP endpoint
+
+Spec **ClusterLogForwarder.obervability.openshift/io** to forward logs to an **OTLP** endpoint
+
+## Summary
+
+The enhancement defines the modifications to the **ClusterLogForwarder** spec 
+and Red Hat log collector to allow administrators to collect and forward logs 
+to an OTLP receiver as defined by the [OpenTelemetry Observability framework](https://opentelemetry.io/docs/specs/otlp/)
+
+## Motivation
+
+Customers continue to look for greater insight into the operational aspects of their clusters by using
+observability tools and open standards to allow them to avoid vedor lock-in.  OpenTelemetry
+
+> ... is an Observability framework and toolkit designed to create and manage telemetry 
+data such as traces, metrics, and logs. Crucially, OpenTelemetry is vendor- and tool-agnostic, meaning 
+that it can be used with a broad variety of Observability backends...as well as commercial offerings.
+
+This framework defines the [OTLP Specification](https://opentelemetry.io/docs/specs/otlp/) for vendors
+to implement.  This proposal defines the use of that protocol to specifically forward logs collected on
+a cluster.  Implementation of this proposal is the first step to embracing a community standard
+and to possibly deprecate the data model currently supported by Red Hat logging.
+
+### User Stories
+
+* As an administrator, I want to forward logs to an OTLP enabled, Red Hat managed LokiStack
+so that I can aggregate logs in my existing Red Hat manged log storage solution.
+**Note:** The realization of this use-case depends on [LOG-XXXX](https://issues.redhat.com/browse/LOG-XXXX)
```

Comment:
```suggestion
* As an administrator, I want to forward logs to an OTLP enabled, Red Hat managed LokiStack
so that I can aggregate logs in my existing Red Hat manged log storage solution.
**Note:** The realization of this use-case depends on [LOG-5523](https://issues.redhat.com/browse/LOG-5523)
```